mongodb-panache-quickstart/src/main/java/org/acme/mongodb/panache/Person.java

```diff
@@ -1,31 +1,28 @@
 package org.acme.mongodb.panache;
 
-import java.time.LocalDate;
-import java.util.List;
-
-import org.bson.codecs.pojo.annotations.BsonProperty;
-
 import io.quarkus.mongodb.panache.MongoEntity;
 import io.quarkus.mongodb.panache.PanacheMongoEntity;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+
+import java.time.LocalDate;
+import java.util.List;
 
 @MongoEntity(collection = "ThePerson")
 public class Person extends PanacheMongoEntity {
     public String name;
-
     // will be persisted as a 'birth' field in MongoDB
     @BsonProperty("birth")
     public LocalDate birthDate;
-
     public Status status;
 
-    // return name as uppercase in the model
+    // return name as lowercase in the model
     public String getName() {
-        return name.toUpperCase();
+        return name.toLowerCase();
     }
 
-    // store all names in lowercase in the DB
+    // store all names in uppercase in the DB
     public void setName(String name) {
-        this.name = name.toLowerCase();
+        this.name = name.toUpperCase();
```

Comment:
Why are you changing this ?

Reply:
Just to avoid to pass the name in capital letters in the URL.

Reply:
OK, then